cmd/kube-apiserver/app/aggregator.go

```diff
@@ -188,6 +188,7 @@ var apiVersionPriorities = map[schema.GroupVersion]priority{
 	{Group: "rbac.authorization.k8s.io", Version: "v1beta1"}:     {group: 17000, version: 12},
 	{Group: "rbac.authorization.k8s.io", Version: "v1alpha1"}:    {group: 17000, version: 9},
 	{Group: "settings.k8s.io", Version: "v1alpha1"}:              {group: 16900, version: 9},
+	{Group: "settings.k8s.io", Version: "v1beta1"}:               {group: 16900, version: 9},
```

Comment:
We are not ready to make v1beta1 higher priority because this change just introduces v1beta1 version so that we can add a few enhancements to it. We will make v1beta1 higher priority when we are done with all the v1beta1 enhancements and it is sufficiently tested.